src/Catty.xcodeproj/project.pbxproj

```diff
@@ -11556,14 +11576,18 @@
 				INFOPLIST_FILE = "Catty/Supporting Files/App-Info.plist";
 				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
+				LIBRARY_SEARCH_PATHS = (
```

Comment:
Why is this change necessary?

Reply:
Sorry this change slipped through, my best guess is its a reminiscence of an old develop branch. I Deleted this change.